components/nodemanager-service/pgdb/node-managers.go

```diff
@@ -339,24 +339,15 @@ func validateNodeManagerFilters(filters []*common.Filter) error {
 		switch filter.Key {
 		case "manager_type":
 			for _, item := range filter.Values {
-				if !isValidManagerType(item) {
-					return &errorutils.InvalidError{Msg: fmt.Sprintf("Invalid manager type filter: %s. manager_type must be one of the following: 'aws-ec2', 'aws-api', 'azure-api'", item)}
+				if !isValidNodeManagerType(item) {
+					return &errorutils.InvalidError{Msg: fmt.Sprintf("Invalid manager type filter: %s. manager_type must be one of the following: 'aws-ec2', 'aws-api', 'azure-api', 'chef', '', 'azure-vm', 'gcp-api' ", item)}
 				}
 			}
 		}
 	}
 	return nil
 }
 
-func isValidManagerType(managerType string) bool {
```

Comment:
we had two functions basically looking at the same thing and i kept getting tripped up on it. figured it's better to just use one!